modules/infrastructure-moving-logging.adoc

```diff
@@ -3,10 +3,85 @@
 // * machine_management/creating-infrastructure-machinesets.adoc
 
 [id="infrastructure-moving-logging_{context}"]
-= Moving the logging aggregation solution
+= Moving the cluster logging resources
 
-[IMPORTANT]
+You can configure the Cluster Logging Operator to deploy the pods 
+for any or all of the Cluster Logging components, Elasticsearch, Kibana, and Curator to different nodes. 
+You cannot move the Cluster Logging Operator pod from its installed location. 
+
+For example, you can move the Elasticsearch pods to a separate node because of 
+high CPU, memory, and disk requirements.
+
+[NOTE]
 ====
-Cluster logging in {product-title} is not installed by default.
-You currently cannot move the logging component to another MachineSet.
+You should set your MachineSet to use at least 6 replicas.
 ====
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed. These features are not installed by default.
+
+* If needed, get the name of the Cluster Logging Custom Resource in the openshift-logging project:
```

Comment:
the clusterlogging CR will always need to be named `instance` fyi. if it isn't the operator will print out a message and not take action on it.